pkgs/public/matrix/matrix-puppet-discord/default.nix

```
@@ -0,0 +1,38 @@
+{ stdenv, pkgs, fetchFromGitHub, nodejs, makeWrapper }:
+
+let
+  nodePackages = import ./node-composition.nix {
+    inherit pkgs;
+    inherit (stdenv.hostPlatform) system;
+    inherit nodejs;
+  };
+in stdenv.mkDerivation rec {
+  pname = "matrix-puppet-discord";
+  version = "0.0.0";
```

This should probably match typical unstable dated version names if it's not a tagged release.
```suggestion
  version = "2020-03-04";
```

---

Good shout